src/components/LandingPage.jsx

```diff
@@ -80,52 +83,57 @@ const LandingPage = ({history}) => {
                         height={300}
                         margin={{left: 10, bottom: 30}}
                         data={data.monthly}
+                        style={{marginLeft: 'auto', marginRight: 'auto'}}
                     >
-                        <Label value="Monthly DOMINO Executions" position="top"/>
-                        <CartesianGrid strokeDasharray="3 3"/>
+                        <CartesianGrid strokeDasharray="3 3" stroke='white'/>
                         <XAxis
                             dataKey="date"
-                            label={{value: 'Date', position: 'bottom'}}
+                            label={{value: 'Date', position: 'bottom', fill: 'white'}}
+                            tick={{ fill: 'white' }} 
                         ></XAxis>
                         <YAxis
-                            label={{value: 'Frequency', angle: -90, position: 'left'}}
+                            label={{value: 'Frequency', angle: -90, position: 'left', fill: 'white'}}
+                            tick={{ fill: 'white' }}
                         />
                         <Tooltip/>
                         <Bar dataKey="freq" fill="#007bff"/> {/*80bdff*/}
                     </BarChart>
                 </Col>
                 <Col>
                     <div className={circle}>{data.total}</div>
-                    <p style={{fontSize: '22px'}}>{attr} this year</p>
+                    <p style={{fontSize: '22px', width: '180px', textAlign: 'center', color: 'white'}}>{attr} this year</p>
                 </Col>
             </Row>
         );
     };
 
     const getCard = (cardDetail) => {
+
+
         switch (cardDetail) {
             case "developerCredits":
-                return DeveloperCredits;
+                return (<DeveloperCreditsCard cardStatus={details=="developerCredits"}/>);
```

Comment:
```suggestion
                return (<DeveloperCreditsCard cardStatus={details==="developerCredits"}/>);
```
Let's try to use the `===` operator if you decide to keep this code.

Reply:
True, but this entire "getCard" function should be removed altogether. Let's discuss this in our next meeting.